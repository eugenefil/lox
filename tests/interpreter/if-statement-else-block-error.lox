if false {
    x;
} else {
    y;
}
