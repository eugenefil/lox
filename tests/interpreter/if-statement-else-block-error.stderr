error: identifier 'y' is not defined
 --> $DIR/if-statement-else-block-error.lox:4:5
  |
4 |     y;
  |     ^
